libs/wingsdk/src/core/inflight.ts

```diff
@@ -44,13 +44,24 @@ export class InflightClient {
     dirname: string,
     filename: string,
     clientClass: string,
-    args: string[]
+    args: string[] | undefined,
+    liftedFields?: Record<string, string>
   ): string {
     const inflightDir = dirname;
     const inflightFile = basename(filename).split(".")[0] + ".inflight";
+    let argsStr = "";
+    if (args !== undefined) {
+      argsStr = args.join(", ");
```

Comment:
can you explain when would we have `args` vs when we'll have `liftedFields`? Are these mutually exclusive?

Reply:
I removed this method (supporting both params was just an interim fix so I could try to validate the initial solution for the PR)